replace-kernel/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This example uses rpm-ostree's cliwrap to allow dracut to run on the container and generate an initramfs.
+FROM quay.io/coreos-assembler/fcos:testing-devel
+# Enable cliwrap.
+RUN rpm-ostree cliwrap install-to-root /
```

Comment:
Is this related?

Reply:
I might have forgoten.. I understood we needed that to get the wrapped kernel-install to be actually used. I'll test to make sure that is the case.

Reply:
Oh duh, of course you're right.

Reply:
That said, I think this requirement is going to be non-obvious for people...we may need to pursue either:

1) Supporting only a subset of cliwrap, e.g. `cliwrap: ["kernel-install"]` in the treefile and then enabling that by default
2) Seeing if we can get a change in `/usr/bin/kernel-install` that will defer to us by default

Reply:
This is a good example of something that we'd get for free if we used the core assembly bits in the container path. It already knows to e.g. ignore kernel scriptlets. (Not suggesting we definitely do this; the added complexity in supporting the container path might not be worth it, but it's worth investigating.)